js/parts/Interaction.js

```diff
@@ -1186,6 +1205,23 @@ extend(Series.prototype, /** @lends Highcharts.Series.prototype */ {
                         i = i + 1;
                     }
                 }
+
+                // For some types (pie, networkgraph, sankey) opacity is
+                // resolved on a point level
+                if (!series.options.inactiveOtherPoints) {
+                    [
+                        series.group,
+                        series.dataLabelsGroup
+                    ].forEach(
+                        function (group) {
+                            if (group) {
+                                group.attr({
```

Comment:
Should be `.animate`. Both this problem and the above (individual handling of markers), can be seen in http://utils.highcharts.local/samples/#view/highcharts/demo/windbarb-series when hovering the wind barbs.

Reply:
Ok 👍

Reply:
Done